src/webgpu/shader/validation/expression/call/builtin/textureDimensions.spec.ts

```diff
@@ -0,0 +1,81 @@
+const builtin = 'textureDimensions';
+export const description = `
+Validation tests for the ${builtin}() builtin.
+
+* test textureDimensions level parameter must be correct type
```

Comment:
Are there tests for the correct return type?

Reply:
no, none of the texture builtins have a test for return type. I guess that needs a todo

Reply:
https://github.com/gpuweb/cts/issues/3586